app/views/EditAry/AssessmentRegistryForm/AdditionalDocument/styles.css

```diff
@@ -0,0 +1,12 @@
+.additional-document {
+    display: grid;
+    grid-template-rows: repeat(2, 50%);
+    grid-template-columns: 1fr 1fr;
+    gap: var(--dui-spacing-extra-large);
+
+    .label-container {
+        color: var(--dui-color-black);
```

Comment:
Don't override colors as much as you can. If the label component is not consistent with the rest of the UI components, talk to @Smritikafle123 to fix this in the mockup.